utils/src/main/kotlin/DirectoryStash.kt

```diff
@@ -26,6 +26,7 @@ import java.io.File
 import java.io.IOException
```

Comment:
Commit message nit: Either "In case of stashing", "When stashing" or "If multiple directories are stashed".

Reply:
The second paragraph could also make use of some general rewording 😉